var/spack/repos/builtin/packages/py-isal/package.py

```diff
@@ -19,3 +19,8 @@ class PyIsal(PythonPackage):
 
     depends_on("python@3.7:", type=("build", "run"))
     depends_on("py-setuptools@51:", type="build")
+    depends_on("py-wheel", type="build")
```

Comment:
Technically wheel is already added by the `PythonPackage` base class so you don't need this line.

Reply:
That was left in while trying to sort out build errors, whoops.